### ISSUE TITLE
 Sorry! Something went wrong. This page didn't load Google Maps correctly

### ISSUE THREAD

**Issue by Vipul-0722** [adrianhajdin/project_travel_advisor#35]
![Screenshot (10223)](https://user-images.githubusercontent.com/90903855/177037075-6681cfd6-5672-41c5-8dcf-e0cc568e2964.png)

**Comment by vidhichadha2507** [comment]
same issue with me as well. 
Someone please help

**Comment by vidhichadha2507** [comment]
<script src="https://maps.googleapis.com/maps/api/js?v=3.exp&libraries=geometry,drawing,places&key="></script>

copy the above script tag and paste it into the index.html file in the public folder. It will work

**Comment by D3troit98** [comment]
I'm having this same issue too. I'm guessing it something to do with billings

**Comment by vidhichadha2507** [comment]
> I'm having this same issue too. I'm guessing it something to do with billings

I don't think so, i just used the script tag mentioned above in index.html and it worked for me.

**Comment by kiritocode1** [comment]
another solution is to reclone the repo , this worked for  ME  , if any problem can i PR your repo ? pwease

**Comment by Vipul-0722** [comment]
> it worked for me.

nope..it is not working for me

**Comment by Vipul-0722** [comment]
@kiritocode1 yess please..

**Comment by yashiikataria** [comment]
Line 6:1:  Expected indentation of 2 space characters but found 0  react/jsx-indent
i am getting this error when i copy pasted it in index.js

> <script src="https://maps.googleapis.com/maps/api/js?v=3.exp&libraries=geometry,drawing,places&key="></script>
> copy the above script tag and paste it into the index.html file in the public folder. It will work

**Comment by tiggy31** [comment]
go to either the console or network tab to see the exact error message you're receiving. Also check index.js to see if React being in strict mode is causing issues.

**Comment by rohitchilhorkar** [comment]
> <script src="https://maps.googleapis.com/maps/api/js?v=3.exp&libraries=geometry,drawing,places&key="></script>
> copy the above script tag and paste it into the index.html file in the public folder. It will work

Thank you! It worked for me
